Models/Article.js

```diff
@@ -0,0 +1,13 @@
+export default class Article {
+  constructor(title, content, writer, likeCount = 0, createdAt = new Date()) {
+    this.title = title;
+    this.content = content;
+    this.writer = writer;
+    this.likeCount = likeCount;
+    this.createdAt = new Date(createdAt); // 현재 시간 저장

```

Comment:
매개변수 자리에 new Date()로 디폴트로 값을 지정 했기 때문에 아래와 같이 지정하면 될 것 같습니다.
기존 코드가 디폴트로 지정한 의미를 상쇄하고 있습니다.
```suggestion
    this.createdAt = createdAt; // 현재 시간 저장

```